### DISCUSSION TITLE
nvim-treesitter installation

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [LazyVim/LazyVim#3945] [Q&A]
After a recent update, I found that `nvim-treesitter` fails to work, so I installed it manually from `https://github.com/nvim-treesitter/nvim-treesitter.git`. But it reports:

>         Origin has changed:
 >         * old: https://github.com/nvim-treesitter/nvim-treesitter.git
 >         * new: https://github.com/neovim/nvim-treesitter.git
 >       Please run update to fix

But, in fact, there is NO `https://github.com/neovim/nvim-treesitter.git`. How to fix it?

**Comment by dotfrag** [comment]
Can you share a way to replicate this? I'm not seeing this locally.
